### PR TITLE
refactor: small fixes for cisco-voice code

### PR DESCRIPTION
**Pull request by laf**
#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/)
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

Just some tidying up. Maybe @adaniels21487 could test / peer review?

### CODE REVIEW

File: includes/polling/cisco-voice/cisco-iosdsp.inc.php [hunk 1]
```diff
@@ -14,32 +14,35 @@
...
-    }
-
-    if (isset($total) && ($total != "") && ($total != 0)) {
-        // Active
-        $active = 0;
-        foreach (snmpwalk_cache_oid_num($device, "1.3.6.1.4.1.9.9.86.1.2.1.1.7", null) as $key => $value) {
-            $active += $value[''];
+    $output = snmpwalk_cache_oid_num($device, "1.3.6.1.4.1.9.9.86.1.2.1.1.6", null);
+    if (is_array($output)) {
+        foreach ($option as $key => $value) {
```
Comment (adaniels21487) [inline includes/polling/cisco-voice/cisco-iosdsp.inc.php]: This should be:
foreach (**$output** as $key => $value) {

File: includes/polling/cisco-voice/cisco-iosxcode.inc.php [hunk 2]
```diff
@@ -16,7 +16,7 @@
     $total = snmpwalk_cache_oid_num($device, "1.3.6.1.4.1.9.9.86.1.7.1.0", null);
     $total = $total['1.3.6.1.4.1.9.9.86.1.7.1.0'][''];
 
-    if (isset($total) && ($total != "") && ($total != 0)) {
+    if (isset($total) && $total > 0) {
         // Available
         $available = snmpwalk_cache_oid_num($device, "1.3.6.1.4.1.9.9.86.1.7.2.0", null);
         $available = $available['1.3.6.1.4.1.9.9.86.1.7.2.0'][''];
```
Comment (adaniels21487) [inline includes/polling/cisco-voice/cisco-iosxcode.inc.php]: You didnt change it, but down below I spot a bug:

dat_update($device, 'cisco-iosxcode', $tags, $fields);
should be:
**data_update**($device, 'cisco-iosxcode', $tags, $fields);

No one will be hitting this bug as on the calling code, the include is wrong:
includes/polling/cisco-voice.inc.php

include "cisco-voice/cisco-xcode.inc.php";
should be:
include "cisco-voice/cisco-**iosxcode**.inc.php";

**Comment by laf** [comment]
Thanks for the review @adaniels21487. Fixed those items now.

**Comment by laf** [comment]
rebased

**Comment by laf** [comment]
Rebased

**Comment by laf** [comment]
bump

**Comment by adaniels21487** [comment]
Hi @laf
I approved this a few days ago, does this still need something from me?

Thanks,
Aaron

**Comment by laf** [comment]
Nope, just needs someone to merge it :)

**Pull request merged**